src/sharedData/components/SharedDataEntryFile.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+
+import { filesize } from 'filesize';
+import _ from 'lodash/fp';
+import { useTranslation } from 'react-i18next';
+import { Link as RouterLink } from 'react-router-dom';
+
+import FileDownloadIcon from '@mui/icons-material/FileDownload';
+import MapIcon from '@mui/icons-material/Map';
+import {
+  Divider,
+  Grid,
+  IconButton,
+  Link,
+  ListItem,
+  Stack,
+} from '@mui/material';
+
+import { formatDate } from 'localization/utils';
+
+import { useGroupContext } from 'group/groupContext';
+import { useSharedData } from 'sharedData/sharedDataHooks';
+
+import SharedDataEntryBase, { ICON_SIZE } from './SharedDataEntryBase';
+import SharedFileIcon from './SharedFileIcon';
+
+const StackRow = props => (
+  <Stack direction="row" alignItems="center" spacing={1} {...props} />
+);
+
+const Visualizations = props => {
+  const { baseOwnerUrl } = useGroupContext();
+  const { i18n, t } = useTranslation();
+  const { file } = props;
+  if (_.isEmpty(file.visualizations)) {
+    return null;
+  }
+
+  return (
+    <Stack
+      component="ul"
+      sx={{ width: '100%', listStyle: 'none', p: 0 }}
+      divider={<Divider component="li" />}
```

Comment:
question: should this divider be `aria-hidden`?

Reply:
yes we should, I will add that